src/fsharp/vs/IncrementalBuild.fs

```diff
@@ -1163,6 +1166,26 @@ type IncrementalBuilder(frameworkTcImportsCache: FrameworkImportsCache, tcConfig
         let flags = tcConfig.ComputeCanContainEntryPoint(sourceFiles |> List.map snd)
         (sourceFiles,flags) ||> List.map2 (fun (m,nm) flag -> (m,nm,flag))
 
+    let defaultTimeStamp = DateTime.Now
```

Comment:
Sorry for being late, but using DateTime.UtcNow everywhere not only a faster way, but mitigates UTC adjustments problems https://blogs.msdn.microsoft.com/kirillosenkov/2012/01/10/datetime-utcnow-is-generally-preferable-to-datetime-now/